packages/ui-bindings/EconomicEvent/AllEconomicEvents.tsx

```diff
@@ -0,0 +1,48 @@
+import { connect } from "react-redux"
+import { gql, graphql, compose } from "react-apollo"
+import { AppState } from "@vflows/store/types"
+import { getActiveLoginToken } from "@vflows/store/selectors/auth"
+
+const query = gql`
+query($token: String) {
+  viewer(token: $token) {
+    allEconomicEvents {
+      id
```

Comment:
This isn't all of the economic event fields. please make a fragment to query all properties with minimal code duplication

Reply:
Fragment added, and properties displayed in the ui
:+1: